corehq/apps/export/views/edit.py

```diff
@@ -31,7 +31,8 @@ def export_id(self):
     def new_export_instance(self):
         return self.export_instance_cls.get(self.export_id)
 
-    def get_export_instance(self, schema, original_export_instance, load_deprecated=False):
+    def get_export_instance(self, schema, original_export_instance):
+        load_deprecated = self.request.GET.get('load_deprecated') == 'True'
```

Comment:
Nit: I'm getting a little nervous seeing a potential `None` == 'True' situation. Maybe default the `load_deprecated` variable to 'False'?

```suggestion
        load_deprecated = self.request.GET.get('load_deprecated', 'False') == 'True'
```

Reply:
In that particular scenario, it would simply revert to `False` as `None != 'True'`. But I do think your suggestion would help make it more readable as to what `load_deprecated` would default to if the condition is not matched. I have made a commit for it :+1: